syntax = "proto2";
import "game_commands.proto";
message Command_SetCardCounter {
    extend GameCommand {
        optional Command_SetCardCounter ext = 1014;
    }
    optional string zone = 1;
    optional sint32 card_id = 2 [default = -1];
    optional sint32 counter_id = 3 [default = -1];
    optional sint32 counter_value = 4;
}
